ql/src/java/org/apache/hadoop/hive/ql/exec/FunctionRegistry.java

```diff
@@ -523,7 +523,6 @@
     system.registerGenericUDF("from_utc_timestamp", GenericUDFFromUtcTimestamp.class);
     system.registerGenericUDF("to_utc_timestamp", GenericUDFToUtcTimestamp.class);
 
-    system.registerGenericUDF("unix_timestamp", GenericUDFUnixTimeStamp.class);
```

Comment:
please don't remove this - instead reference the remaining funciton

Reply:
after this you could probably undo some of the unnecessary changes to `to_unix_timestamp` in the tests